hibernate-core/src/main/java/org/hibernate/cfg/PersistenceSettings.java

```diff
@@ -182,4 +182,12 @@ public interface PersistenceSettings {
 	 */
 	@Deprecated
 	String JPA_TRANSACTION_TYPE = "javax.persistence.transactionType";
+
+	/**
+	 * Specifies whether unowned (i.e. {@code mapped-by}) associations should be considered
+	 * when validating transient entity instance references.
+	 *
+	 * @settingDefault {@code false}
+	 */
+	String STRICT_UNOWNED_TRANSIENT_CHECK = "hibernate.strict_unowned_transient_check";
```

Comment:
Open to suggestions on the naming

Reply:
Suggestion: `hibernate.unowned_association_integrity_checking`

Reply:
I think we should have `transient` somewhere in there, maybe `unowned_association_transient_check`?